salt/orch/force-removal.sls

```diff
@@ -39,6 +39,11 @@ remove-cluster-wide-removal-grain:
     - kwarg:
         destructive: True
 
+remove-target-mine:
+  salt.function:
+    - tgt: '{{ target }}'
```

Comment:
Is this run in the `target` or in the master? This could be problematic if it runs in the minions and it is not responding....

Reply:
This is ran on the minion. Nothing depends on this state, so it shouldn't be a problem, but will update with `fail_minions` so we are sure that failure on `target` is an option. Thanks!